Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -548,10 +549,11 @@ withStmt :
     statementLabelDefinition? whiteSpace? END_WITH
 ;
 
-// Special forms with special syntax, only available in a report.
+// Special forms with special syntax, only available in VBA reports or VB6 forms and pictureboxes.
 lineSpecialForm : expression whiteSpace (STEP whiteSpace?)? tuple MINUS (STEP whiteSpace?)? tuple whiteSpace? (COMMA whiteSpace? expression)? whiteSpace? (COMMA whiteSpace? lineSpecialFormOption)?;
 circleSpecialForm : (expression whiteSpace? DOT whiteSpace?)? CIRCLE whiteSpace (STEP whiteSpace?)? tuple (whiteSpace? COMMA whiteSpace? expression)+;
 scaleSpecialForm : (expression whiteSpace? DOT whiteSpace?)? SCALE whiteSpace tuple whiteSpace? MINUS whiteSpace? tuple;
+pSetSpecialForm : (expression whiteSpace? DOT whiteSpace?)? PSET (whiteSpace STEP)? whiteSpace? tuple whiteSpace? (COMMA whiteSpace? expression)?;
```

Comment:
since PSET is also registered as a keyword, it seems my concern about this needing to be added to something like `expression` is unnecessary